home-page.component.html

```diff
@@ -0,0 +1,32 @@
+<h1>Local weather</h1>
+<section>
+    <div class="fact-weather">
+        <div class="title">Actual weather</div>
+        <div class="city">
+            <span class="city-title"> {{this.weatherData.forCity}}</span>

```

Comment:
You can use access to property without `this`. Please, check all code

Reply:
I removed "this" from all the places where it was nacessary
link: https://github.com/KatrinSha/Weather/pull/2/commits/df854221f59f3a1fa1016069fa2c8a3f98270111